src/mono/wasm/wasm.proj

```diff
@@ -158,7 +158,7 @@
     <PropertyGroup>
       <_EmccExportedRuntimeMethods>&quot;[@(EmccExportedRuntimeMethod -> '%27%(Identity)%27', ',')]&quot;</_EmccExportedRuntimeMethods>
       <_EmccExportedFunctions>@(EmccExportedFunction -> '%(Identity)',',')</_EmccExportedFunctions>
-      <EmccInitialHeapSize>536870912</EmccInitialHeapSize>
+      <EmccInitialHeapSize>16777216</EmccInitialHeapSize>
```

Comment:
what happens when we are not (re)linking? would that value be sufficient in most cases?

Reply:
or maybe that's just for the dotnet.wasm for workload. do we need to set it then here?

Reply:
We have 16MB default for Net7 and this is only putting it back.
The memory could always grow from there, the only cost would be the re-allocation.

Reply:
> or maybe that's just for the dotnet.wasm for workload. do we need to set it then here?

This is for the dotnet.wasm of "preview" - no re-link. 
If we don't set it here , current version of emcc will pass the same value.
I think it's ok to have it explicit.